crates/router/src/routes/dummy_connector/utils.rs

```diff
@@ -0,0 +1,98 @@
+use app::AppState;
+use error_stack::{IntoReport, ResultExt};
+use masking::ExposeInterface;
+use rand::Rng;
+use redis_interface::RedisConnectionPool;
+use tokio::time as tokio;
+
+use super::{
+    errors::DummyConnectorErrors,
+    types::{
+        DummyConnectorPaymentData, DummyConnectorPaymentMethodData, DummyConnectorPaymentRequest,
+        DummyConnectorPaymentResponse, DummyConnectorResponse, DummyConnectorTransactionStatus,
+    },
```

Comment:
consider using non-specific imports, an example would be:
```suggestion
    errors, types
```

and then instead of writing the type in the code below write it relative to the sub-module, i.e. `types::DummyConnectorPaymentData`